build/__dotNetCoreRunTimeVersions.sh

```diff
@@ -1,6 +1,6 @@
 # This file was auto-generated from 'constants.yaml'. Changes may be overridden.
 
-DOT_NET_CORE_RUNTIME_BASE_TAG='20200331.2'
+DOT_NET_CORE_RUNTIME_BASE_TAG='20200424.2'
```

Comment:
How about other platforms' base tags, the ones in `_nodeVersion.sh`, `_phpVersion.sh` and `_pythonVersion.sh`? I saw they are not updated to most recent ones yet, maybe update in a new PR?..

Reply:
Sure sounds good.